spec/compute_spec.rb

```diff
@@ -0,0 +1,29 @@
+require 'spec_helper'
+require_relative './shared_context'
+
+require 'fog/kubevirt/compute/compute'
+
+describe Fog::Kubevirt::Compute do
+  before :all do
+    vcr = KubevirtVCR.new(
+      :vcr_directory => 'spec/fixtures/kubevirt/service',
```

Comment:
In terms of folder structure, let's have it organized by kubevirt's version.
In that case we'll be able to test and make should api compatibility.

Reply:
Ok, good idea.